test/test_manifest.json

```diff
@@ -7292,5 +7292,13 @@
       "rounds": 1,
       "link": true,
       "type": "eq"
+   },
+   {
+      "id": "issue15289",
+      "file": "pdfs/issue15289.pdf",
+      "md5": "f3d20aee398e052ac3dce7cc030f50de",
+      "rounds": 1,
+      "lastPage": 1,
+      "type": "eq"
```

Comment:
There's already a test-case for this document, hence you'll need to remove all the changes to the `test/`-folder in this patch.